pinot-broker/src/main/java/org/apache/pinot/broker/api/RequestStatistics.java

```diff
@@ -118,6 +128,14 @@ public void setBrokerId(String brokerId) {
     _brokerId = brokerId;
   }
 
+  public void setOfflineBrokerTenant(String offlineBrokerTenant) {
```

Comment:
We don't need broker tenant names. We need offline and realtime server tenant names

Reply:
done